org.geeqie.Geeqie.json

```diff
@@ -0,0 +1,134 @@
+{
+    "app-id": "org.geeqie.Geeqie",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "geeqie",
+    "rename-desktop-file": "geeqie.desktop",
+    "rename-icon": "geeqie",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2 -std=c++11",
+        "env": {
+            "V": "1"
+        }
+    },
```

Comment:
I don't think you need any of that, most of them are set by default.

Reply:
True, I removed them

Reply:
True, I removed them now.